tests/tlm/conftest.py

```diff
@@ -47,6 +47,10 @@ def tlm_dict(studio: Studio) -> Dict[str, Any]:
             tlm_dict[quality_preset][model]["tlm"] = studio.TLM(
                 quality_preset=quality_preset, options=options
             )
+            tlm_dict[quality_preset][model]["tlm_no_options"] = studio.TLM(
```

Comment:
Can you add a comment/docstring here about why we are instantiating 2 TLM objects here? Something about wanting to test both the random options + default options